contrib/deploy-gitea/01-namespace.yml

```diff
@@ -0,0 +1,4 @@
+kind: Namespace
```

Comment:
Why do we need gitea? Maybe I'm wrong or missing something but I think we don't need argocd to deploy it

Reply:
gitea is used for organizations that do not have access to github.com or bitbucket to stand up a repo and point to this one https://github.com/Red-Hat-SE-RTO/openshift-ztp. There are some users that have proxy issues or firewall issues that do not allow them to access git repo.